controllers/apply/form-router-next/lib/application-expiry.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+const path = require('path');
+const get = require('lodash/get');
+const moment = require('moment');
+const config = require('config');
+const enableExpiration = config.get('awardsForAll.enableExpiration');
+
+const { sendHtmlEmail } = require('../../../../common/mail');
+const {
+    PendingApplication,
+    ApplicationEmailQueue
+} = require('../../../../db/models');
+
+const { getEmailFor, getPhoneFor } = require('../../../../common/contacts');
+
+const {
+    signTokenUnsubscribeApplicationEmails
```

Comment:
Can we inline this code? Having a method called `signTokenUnsubscribeApplicationEmails` in common feels too specific, is it only related to this logic?